building_energy_standards_data/applications/create_openstudio_standards_json.py

```diff
@@ -340,6 +341,7 @@ def create_openstudio_standards_data_json_ashrae_90_1(
         code_version=version_90_1,
         template=f"90.1-{version_90_1}",
         tables_to_export=tables_to_export_90_1,
+        osstd_repository_path=osstd_repository_path,
```

Comment:
Prior calling this function, I suggest do a check to make sure the user provided `osstd_repository_path` is a valid path.
We can handle it through Exceptions (`OpenStudioStandardsDataException`).

Reply:
Addressed.